CarbonChat-API/src/main/java/net/draycia/carbon/api/channels/ChatChannel.java

```diff
@@ -15,46 +14,49 @@
 
 public abstract class ChatChannel implements ForwardingAudience {
```

Comment:
Why isn't ChatChannel an interface? It contains no state, and the one implemented method could just as well be a default interface method

Reply:
168233d